data/categories.txt

```diff
@@ -14863,6 +14863,46 @@ zh-Hant:國家公園
 fa:پارک ملی
 mr:राष्ट्रीय उद्यान
 
+boundary-aboriginal_lands
+en:Indigenous Lands
```

Comment:
Are you sure that someone will search for added boundaries by typing these search terms? Do we really need these entries in categories.txt @vng ?

Reply:
tbh I am unsure either way. If someone doesn't want them included then I am fine with removing them

Reply:
If we leave it searchable then additional synonyms should be added that may be likely typed by users: aboriginal, indians, etc.

Reply:
Good point, I guess I will add those for English since its a native language of mine.
In English I at least know that there are still many people calling themselves "American Indian" or "Aborigine" so those are fine. (it really depends on who you ask, some are neutral about it while others preferred to be called by one of these three collective terms)

For Chinese I will only add alternate words for "territory" and "land(s)" though since the many alternate words
for "Indigenous" have been rejected by Taiwanese Indigenous people in favour of a term they came up
with themselves：「原住民」。(A quick search brings up the [English](https://en.wikipedia.org/wiki/Taiwanese_indigenous_peoples#Indigenous_political_movement) and [Chinese](https://zh.wikipedia.org/wiki/%E8%87%BA%E7%81%A3%E5%8E%9F%E4%BD%8F%E6%B0%91%E6%97%8F#%E6%88%B0%E5%BE%8C%E6%99%82%E6%9C%9F) wikipedia pages supporting this notion)